goal_src/jak3/engine/common-obs/generic-obs.gc

```diff
@@ -4050,9 +4047,9 @@
   :virtual #t
   :trans (behavior ()
     (set! (-> self task-counter) (-> *game-info* task-counter))
-    (if *bigmap*
-        (bigmap-method-16 *bigmap*)
-        )
+    ;; (if *bigmap*
+    ;;     (bigmap-method-16 *bigmap*)
+    ;;     )
```

Comment:
maybe this could be `(if (and *bigmap* (nonzero? *bigmap*))` so it doesn't do something wrong when we forget to put it back?